### DISCUSSION TITLE
Suggestion: Show QSO listing when clicking on cell in Awards window

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#166] [Ideas]
Hi Ladislaw,
The Awards View is nice. To make it more usable I suggest you add a ferature to show the qsos related to each cell  in the Country view. This would help the operator with the bookeeping on QSL status. 

The List of QSOs could be shown in a new pop up window, or In the Log list view already present:

1. Click on Country name or Prefix: Show all QSOs with the entity related to the overall search criteria in options menu. 
2. Click on Cell related to Band: Show all QSOs with the entity on band related to the overall search criteria in options menu. 

73 de LB2EG Richard

![Screenshot from 2023-01-22 14-34-19](https://user-images.githubusercontent.com/22942440/213918901-8f995afa-894a-4e93-9aec-414948531bb2.png)

**Comment by rsaue** [comment]
Hi Ladislaw,.
I have had a sneak peek on the testing_0.19 branch. 

I like the listing of qsos related to the cell in awards view (and I really like the aurora on the online map!) 

I suggest  you add a way to reset the selection in the search menu for the Log list when entering a new callsign in he new QSO Callsign field, giving the Log list a fresh start with the new QSO. 

My point is shown in the attached screenshot. There I have prevously been in  Awards View, and selected the cell related to Aland islands on 15 meter band. Afterwards I have in Dx cluster window klicked on LA1MFA, whose call sign is then filled into the QSO callsign field and in the Search field for the Log list. 

I have several QSOs with LA1MFA in log. But nothings shows up in the Log list because the band and the country field is not reset. It is still on 15 meters and Aland islands 

![Screenshot from 2023-01-26 23-33-53](https://user-images.githubusercontent.com/22942440/214965971-dc21786f-2646-474c-9a5e-b25b8dc9319d.png)

73 de LB2EG Richard

**Reply by foldynl** [reply]
You're right. I came across this while testing and the filter behavior after Award Dialog Exit also bothered me. But in this case, I'm not sure when exactly the reset should occur. At first I thought that it was necessary to perform a reset when closing the Awards dialog. But then it occurred to me. Isn't it possible that the operator clicked on Awards and wanted to make a selection and then continue with this selection despite the fact that he closed the Awards window? Another question is whether it should be reset to default (no filter active) or it should be returned to its original state (meaning the state before opening the Awards dialog). I am still tuning this new feature. What is in the testing branch is certainly not the final form.

thank you very much for your feedback.

**Reply by rsaue** [reply]
Hi, 
What about having a "Reset" button to the right of the "user filter"  in search menu?

73 de LB2EG Richard

**Reply by foldynl** [reply]
Try the latest testing_0.19 branch. It's already been resolved. The last settings are saved, even if QLog is restarted.

**Reply by rsaue** [reply]
Hi, 
Sorry, but i cannot find any change. When I exit the Awards view the filter criteria does not to no selection, and when i enter a cal in qso logging field, the selection from Awards view is still there. .... 

73 de LB2EG Richard

**Reply by foldynl** [reply]
Did you pull a fresh testing0.19? Unfortunately, I cannot confirm what you see. See example below. "callsign" search is not saved because it is deleted when new QSO callsign is entered.
![test](https://user-images.githubusercontent.com/942769/216113133-78594b78-543b-4a84-8362-4c21eb6324e6.gif)

**Reply by foldynl** [reply]
I'm sorry, there is really an issue. It only happens only when QLog is run for the first time in version 0.19.

**Reply by foldynl** [reply]
it should  work now.

**Reply by rsaue** [reply]
Hi, 
Yes, works now after I restarted QLog. 

73 de LB2EG Richard

**Reply by foldynl** [reply]
many thanks for your tests
